### PR TITLE
chore(devops): docker cmds

### PR DESCRIPTION
**Pull request by sdaveas**
handle mnemonic commands in `entrypoint.sh` for seamless integration with axelarate.

For now, _we don't_ forward user's password. This will be patched after closing axelarnetwork/axelarate#269.

### CODE REVIEW

**Comment by sdaveas** [comment]
Duplicated PR with https://github.com/axelarnetwork/tofnd/pull/194. Changes lost due to bad rebase.

File: docker-compose.yml [hunk 1]
```diff
@@ -10,7 +15,5 @@ services:
     volumes:
       - tofnd:/.tofnd
     environment:
-      # ATTENTION: use --unsafe only for testing
-      - UNSAFE=true
-      # available cmds: create (default), stored, update, import, export
-      - MNEMONIC_CMD=create 
+      # available cmds: auto, create, existing (default), import, export
```
Comment (milapsheth) [inline docker-compose.yml:18]: auto is the default here

**Review by milapsheth — APPROVED** [review]
Feel free to merge after addressing the comment
Edit: didn't notice this was already merged 😅

File: docker-compose.yml (hunk 1 above)
Comment (ggutoski) [reply docker-compose.yml:18]: I made the same comment in #194 and it was fixed there then unfixed by the rebase 😛

Comment (sdaveas) [reply docker-compose.yml:18]: I don't understand what happened... I am sure I had these changed commited at #194

Comment (sdaveas) [reply docker-compose.yml:18]: oh I see. If you refer to [this](https://github.com/axelarnetwork/tofnd/pull/194#discussion_r718583329) comment, these changes are committed fine.

**Pull request merged**